LiveStream/ViewModels/LiveStreamViewModel.swift

```diff
@@ -132,7 +132,7 @@ internal final class LiveStreamViewModel: LiveStreamViewModelType, LiveStreamVie
       isMaxOpenTokViewersReached,
 
       liveStreamEvent
-        .map { event in event.stream.isRtmp || didEndNormally(event: event) }
+        .map { event in event.isRtmp ?? false || didEndNormally(event: event) }
```

Comment:
i'm a big fan of doing explicit `event.isRtmp == .some(true)` over coalescing for booleans